src/common/databaseFilters.ts

```diff
@@ -3,57 +3,86 @@ import { displayPostcodeForHomelessClient } from "./format";
 import { DbClientRow, DbParcelRow } from "@/databaseUtils";
 import { parcelsPageDeletedClientDisplayName } from "@/app/parcels/parcelsTable/format";
 
-export const fullNameSearch = <DbData extends DbClientRow | DbParcelRow>(
-    fullNameColumnLabel: Extract<keyof DbData, "full_name" | "client_full_name">,
-    clientIsActiveColumnLabel: Extract<keyof DbData, "is_active" | "client_is_active">
+const textFilterDelimiter = ",";
+
+export const dbFilterWithSubstringQueries = <DbData extends DbClientRow | DbParcelRow>(
```

Comment:
This is the core of this change. 
The special characters allowed in row 14 are the same for all text filters and are the superset of all the special characters that were allowed previously in different cases.